game-core/src/main/java/games/strategy/triplea/settings/ClientSetting.java

```diff
@@ -163,36 +170,23 @@
    * </p>
    */
   public static void initialize() {
-    setPreferences(Preferences.userNodeForPackage(ClientSetting.class));
-  }
-
-  @VisibleForTesting
-  static void setPreferences(final Preferences preferences) {
-    checkNotNull(preferences);
-
-    preferencesRef.set(preferences);
+    setPreferences(
+        Preconditions.checkNotNull(Preferences.userNodeForPackage(ClientSetting.class)));
```

Comment:
Note that `Preferences#userNodeForPackage()` is documented to never return `null`, so `checkNotNull()` here is redundant.

Reply:
This was moved from `setPreferences`, this is the one non-test location that calls that method. That would mean the 'checkNotNull'  was redundant in that method as well.

Reply:
removed in: 25ea84e2d